src/lookup.rs

```diff
@@ -14,6 +11,8 @@ use lazy_static::lazy_static;
 
 use std::{process, str};
 use regex;
+use gc::Gc;
+use crate::scope::Scope;
 
 lazy_static! {
     static ref BUILTINS: Vec<String> = vec![
```

Comment:
(note for later: we should replace this list by reading the keys from `builtins.rs`.